app/components/sidebars/settings/status_label/status_label.js

```diff
@@ -61,7 +61,6 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
     return {
         label: {
             color: changeOpacity(theme.centerChannelColor, 0.5),
-            flex: 1,
```

Comment:
why this change?

Reply:
@enahum I think it might be needed (in a file above the same change occured) for making `justifyContent: 'center'` work correctly in iOS